src/CarbonAware.WebApi/src/Models/CarbonIntensityBatchDTO.cs

```diff
@@ -0,0 +1,9 @@
+namespace CarbonAware.WebApi.Models;
+
+using System.Text.Json.Serialization;
+
+[Serializable]
+public record CarbonIntensityBatchDTO : CarbonIntensityBaseDTO
```

Comment:
NIT: single line to signal that this is purposefully empty

```suggestion
public record CarbonIntensityBatchDTO : CarbonIntensityBaseDTO { }
```